spi/common/aggregate-service-spi/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
```

Comment:
Whut? :D

Reply:
it has been a long way through conflicts :'D